exp/api/v1beta2/awsmanagedmachinepool_types.go

```diff
@@ -120,6 +120,10 @@ type AWSManagedMachinePoolSpec struct {
 	// +optional
 	InstanceType *string `json:"instanceType,omitempty"`
 
+	// InstanceTypes specifies the AWS instance types
+	// +optional
```

Comment:
Can we elaborate in the comment what happens when you specify multiple instances types?
What's the expectation, allocation strategy...?

Reply:
The expectation is any of these instance types could be present in the node group. The allocation strategy is completely managed by AWS. Is there anything you think I should mention in the doc string?

Reply:
I agree that would be a good idea to expand the comment. I quite like the suggestion fro @enxebre :+1: 

> Ordering of the passed values means priority. See https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html#managed-node-group-capacity-types for more info

Reply:
Updated the doc string. PTAL.